diff -urN origsrc//p7zip_15.14/makefile src//p7zip_15.14/makefile
--- origsrc//p7zip_15.14/makefile	2011-03-13 07:52:45.000000000 -0400
+++ src//p7zip_15.14/makefile	2011-06-26 16:25:33.265000000 -0400
@@ -56,6 +56,5 @@
 	$(MAKE) -C CPP/7zip/UI/Client7z           depend
 	$(MAKE) -C CPP/7zip/UI/Console            depend
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree  depend
-	$(MAKE) -C CPP/7zip/Compress/Rar          depend
 	$(MAKE) -C CPP/7zip/UI/GUI                depend
 	$(MAKE) -C CPP/7zip/UI/FileManager        depend
@@ -68,7 +67,6 @@
 common7z:common
 	$(MKDIR) bin/Codecs
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree all
-	$(MAKE) -C CPP/7zip/Compress/Rar         all
 
 lzham:common
 	$(MKDIR) bin/Codecs
@@ -93,7 +91,6 @@
 	$(MAKE) -C CPP/7zip/UI/FileManager       clean
 	$(MAKE) -C CPP/7zip/UI/GUI               clean
 	$(MAKE) -C CPP/7zip/Bundles/Format7zFree clean
-	$(MAKE) -C CPP/7zip/Compress/Rar         clean
 	$(MAKE) -C CPP/7zip/Compress/Lzham       clean
 	$(MAKE) -C CPP/7zip/Bundles/LzmaCon      clean2
 	$(MAKE) -C CPP/7zip/Bundles/AloneGCOV    clean
diff -urN origsrc//p7zip_15.14/makefile.machine src//p7zip_15.14/makefile.machine
--- origsrc//p7zip_15.14/makefile.machine	2011-03-13 07:54:57.000000000 -0400
+++ src//p7zip_15.14/makefile.machine	2011-06-26 16:26:33.595000000 -0400
@@ -4,13 +4,13 @@
 
-OPTFLAGS=-O -s
+OPTFLAGS=-O
 
 ALLFLAGS=${OPTFLAGS} -pipe \
     	-D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE \
 	-DNDEBUG -D_REENTRANT -DENV_UNIX \
 	-D_7ZIP_LARGE_PAGES \
 	$(LOCAL_FLAGS)
 
-CXX=g++
+CXX=g++ -DHAVE_GCCVISIBILITYPATCH -fvisibility=hidden -fvisibility-inlines-hidden
 CC=gcc
 CC_SHARED=-fPIC
 LINK_SHARED=-fPIC -shared
