diff -urN origsrc/p7zip_9.38.1/CPP/Common/StringConvert.cpp src/p7zip_9.38.1/CPP/Common/StringConvert.cpp
--- origsrc/p7zip_9.38.1/CPP/Common/StringConvert.cpp	2015-02-25 11:35:04.000000000 -0800
+++ src/p7zip_9.38.1/CPP/Common/StringConvert.cpp	2015-06-09 21:04:52.578431100 -0700
@@ -178,7 +178,7 @@
     }
   }
 #else
-  UString &srcString = src;
+  const UString &srcString = src;
 #endif
 
   if ((global_use_utf16_conversion) && (!srcString.IsEmpty()))
