--- origsrc/p7zip_15.09/CYGWIN-PATCHES/setup.hint	1969-12-31 19:00:00.000000000 -0500
+++ src/p7zip_15.09/CYGWIN-PATCHES/setup.hint	2011-06-26 17:43:43.995000000 -0400
@@ -0,0 +1,8 @@
+category: Archive
+requires: bash libgcc1 libstdc++6
+sdesc: "A file archiver with very high compression ratios."
+ldesc: "p7zip is the Unix port of 7-Zip, a file archiver that archives with
+very high compression ratios.  It supports packing/unpacking 7z, zip, gzip
+bzip2, xz, tar, and wim archives.  It supports unpacking of arj, cab, chm,
+cpio, dmg, hfs, lzh, lzma, wim, xar, z, nsis, deb, rpm, msi, udf, iso, ntfs,
+fat, vhd, and mbr archives.  This version does NOT support rar archives."
