--- origsrc/p7zip_9.20.1/CYGWIN-PATCHES/README	1969-12-31 19:00:00.000000000 -0500
+++ src/p7zip_9.20.1/CYGWIN-PATCHES/README	2011-06-26 17:43:43.988000000 -0400
@@ -0,0 +1,171 @@
+p7zip
+========================================================================
+p7zip is the Unix port of 7-Zip, a file archiver that archives with very
+high compression ratios.
+
+p7zip provides /usr/bin/7zr, a standalone minimal version of the 7-zip
+tool that only handles 7z archives.  It also provides /usr/bin/7za, a
+statially-linked application that supports packing and unpacking 7z,
+zip, gzip, bzip2, and tar archives. Additionally, it provides
+/usr/bin/7z, a module-based application that supports the full set of
+7-zip compression formats:
+  pack/unpack: 7z, zip, gzip, bzip2, xz, tar, wim
+  unpacking:   arj, cab, chm, cpio, deb, dmg, hfs, lzh, lzma, wim, xar, z
+               installer formats: nsis, deb, rpm, msi
+               filesystem images: cramfs, squashfs, udf, iso, ntfs, fat,
+	                          vhd, mbr
+The modules used by 7z are located in /usr/lib/p7zip/.
+
+RAR unpacking support is NOT provided, as the UnRAR license not
+OSI-approved and is incompatible with cygwin.
+
+Runtime requirements
+  cygwin-1.7.9-1
+  libgcc1-4.3.4-4
+  libstdc++6-4.3.4-4
+
+Build requirements
+  binutils-2.20.51-2
+  gcc-core-4.3.4-4
+  gcc-g++-4.3.4-4
+  bash-4.1.10-4
+  make-3.81-2
+  nasm-2.08.02-1
+  cygport-0.10.4-1
+
+Canonical homepage:
+  http://www.7-zip.org/
+  http://p7zip.sourceforge.net/
+
+Canonical download:
+  http://sourceforge.net/project/showfiles.php?group_id=111810
+
+License:
+  LGPL, BSD-no-advert
+  + RAR code is non-OSI approved, and is not compiled
+    into these distributed binaries.
+
+Language:
+  C, C++
+
+========================================================================
+
+Build instructions:
+  unpack p7zip-9.20.1-X-src.tar.bz2
+    if you use setup to install this src package, it will be unpacked
+    under /usr/src automatically
+  cd /usr/src
+  cygport ./p7zip-9.20.1-X.cygport all
+
+This will create:
+  /usr/src/p7zip-9.20.1-X.tar.bz2
+  /usr/src/p7zip-9.20.1-X-src.tar.bz2
+
+========================================================================
+
+Files included in p7zip-9.20.1-X:  see cygcheck -l output
+
+========================================================================
+
+Port Notes:
+----------  p7zip-9.20.1-1 -- 26 Jun 2011 -----------
+* updaste to latest release
+* compile using gcc/g++ 4.3.4
+* Notable upstream changes:
+  = From Windows version of 7-zip
+    - 7-Zip now supports LZMA2 compression method.
+    - 7-Zip now can update solid .7z archives.
+    - 7-Zip now supports XZ archives.
+    - 7-Zip now supports PPMd compression in ZIP archives.
+    - 7-Zip now can unpack NTFS, FAT, VHD, MBR, APM, SquashFS, CramFS, MSLZ archives.
+    - 7-Zip now can unpack GZip, BZip2, LZMA, XZ and TAR archives from stdin.
+    - 7-Zip now can unpack some TAR and ISO archives with incorrect headers.
+    - 7-Zip now supports files that are larger than 8 GB in TAR archives.
+    - NSIS and WIM support was improved.
+    - Partial parsing for EXE resources, SWF and FLV.
+    - The support for archives in installers was improved.
+    - 7-Zip now can stores NTFS file timestamps to ZIP archives.
+    - Speed optimizations in PPMd codec.
+    - Speed optimizations in CRC calculation code for Intel's Atom CPUs.
+    - New -scrc switch to calculate total CRC-32 during extracting / testing.
+    - 7-Zip File Manager now doesn't use temp files to open nested archives stored without compression.
+    - Disk fragmentation problem for ZIP archives created by 7-Zip was fixed.
+* Forward ported existing debian patchset
+  01_makefile_9.20.1.patch              : don't compile non-free RAR code
+  02_man_9.20.1.patch                   : format corrections for nroff
+  03_respect-umask-partial_9.20.1.patch
+  SEE: debian notes concerning directory exclusion, in
+       p7zip-4.47-1 section below
+ 
+----------  p7zip-9.04-10 -- 04 Aug 2009 -----------
+* update to latest release
+* Notable upstream changes:
+  = From Windows version of 7-zip 9.04 :
+    - 7-Zip now can update solid .7z archives.
+    - 7-Zip now supports LZMA2 compression method.
+    - 7-Zip now supports XZ archives.
+    - 7-Zip now can unpack NTFS, FAT, VHD and MBR archives.
+    - 7-Zip now can unpack GZip, BZip2, LZMA, XZ and TAR archives from stdin.
+    - New -scrc switch to calculate total CRC-32 during extracting / testing.
+    - Some bugs were fixed.
+  = Fixed #2799966 "A newly created 7z archive (by p7zip 4.65) is broken and
+    cannot be unpacked / listed / tested"
+* update setup.hint: No longer explicitly specify requires: cygwin, but do
+  specify bash dependency for p7zip wrapper script.
+* update to latest debian patchset
+  01_makefile.patch             : don't compile non-free RAR code
+  02_man.patch                  : format corrections for nroff
+  03_respect-umask-partial.diff : Most of original 03 patch absorbed
+                                  upstream. (Ditto 04 patch)
+  SEE: debian notes concerning directory exclusion, in
+       p7zip-4.47-1 section below
+
+----------  p7zip-4.57-10 -- 04 Apr 2009 -----------
+* Fork for cygwin-1.7 development
+* Slight differences in documentation installation paths
+  to align with cygwin-1.7 standards.
+
+----------  p7zip-4.57-1 -- 04 Apr 2009 -----------
+* update to latest release
+* update to latest debian patchset
+  01_makefile.patch             : don't compile non-free RAR code
+  02_man.patch                  : format corrections for nroff
+  03_respect-umask-partial.diff : Most of original 03 patch absorbed
+                                  upstream. (Ditto 04 patch)
+  SEE: debian notes concerning directory exclusion, in
+       p7zip-4.47-1 section below
+
+----------  p7zip-4.57-1 -- 04 Mar 2008 -----------
+* update to latest release
+* update to new cygport facilities
+* update to latest debian patchset
+    01_makefile.patch           : don't compile non-free RAR code
+    02_man.patch                : format corrections for nroff
+    03_respect-umask.diff       : self explanatory
+    04_no_strip_by_default.diff : only affects linux
+  SEE: debian notes concerning directory exclusion, in
+       p7zip-4.47-1 section below
+* some updates to p7zip script
+
+----------  p7zip-4.47-2 -- 20070617 -----------
+* fix packaging nits
+
+----------  p7zip-4.47-1 -- 20070615 -----------
+* initial cygwin packaging
+* incorporated debian patches from
+  http://packages.qa.debian.org/p/p7zip.html:
+    01_makefile:      don't compile non-free RAR code
+    02_man:           format corrections for nroff
+    03_respect-umask: self explanatory
+    p7zip.1 manpage copied directly
+    some updates to the existing p7zip script
+* debian notes concerning directory exclusion.
+  ------------------------------------
+  - Directory exclusion only works when an exclude list file is provided
+    and if directories to exclude are listed as relative paths of the
+    location 7z or 7za is run from.
+  - Using absolute directories *won't* work, as noted in debian
+    bug #296665.
+
+==============================================================
+Cygwin port maintained by: Chuck Wilson <cygwin@cygwin.com>
--- origsrc/p7zip_9.20.1/CYGWIN-PATCHES/setup.hint	1969-12-31 19:00:00.000000000 -0500
+++ src/p7zip_9.20.1/CYGWIN-PATCHES/setup.hint	2011-06-26 17:43:43.995000000 -0400
@@ -0,0 +1,8 @@
+category: Archive
+requires: bash libgcc1 libstdc++6
+sdesc: "A file archiver with very high compression ratios."
+ldesc: "p7zip is the Unix port of 7-Zip, a file archiver that archives with
+very high compression ratios.  It supports packing/unpacking 7z, zip, gzip
+bzip2, xz, tar, and wim archives.  It supports unpacking of arj, cab, chm,
+cpio, dmg, hfs, lzh, lzma, wim, xar, z, nsis, deb, rpm, msi, udf, iso, ntfs,
+fat, vhd, and mbr archives.  This version does NOT support rar archives."
